portfolio-project-matching-app/components/FilterButtons.js

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react'
+import Button from './Button'
+
+const FilterButtons = ({ category, choices, onClick }) => {
```

Comment:
A ton of how this component interacts with its parent component is `todo`. When a filter/sort button is selected/deselected, an ancestor's `onClick` is engaged. I'm sure as we integrate this into the bigger picture, we can come up with some more descriptive names.